skema/text_reading/text_reading/src/main/scala/org/ml4ai/skema/text_reading/grounding/GrounderFactory.scala

```diff
@@ -2,25 +2,31 @@ package org.ml4ai.skema.text_reading.grounding
 
 import com.typesafe.config.Config
 import org.clulab.processors.fastnlp.FastNLPProcessor
+import scala.collection.JavaConverters._
 
 object GrounderFactory {
   def getInstance(config: Config, chosenEngine: Option[String] = None): Grounder = {
     val engine = chosenEngine getOrElse config.getString("engine")
     val prependManualGrounder = config.getBoolean("forceManualGroundings")
-    lazy val manualGrounder = buildManualGrounder(config)
+    val domain = config.getString("domain")
+    val domainConfig = config.getConfig(domain)
+
+    lazy val manualGrounder = buildManualGrounder(domainConfig)
     // Grounding parameters
     engine.toLowerCase match {
         case "miraembeddings" =>
-          val ontologyFilePath = config.getString("ontologyPath")
-          val lambda = config.getInt("lambda")
-          val alpha = config.getDouble("alpha").toFloat
-          val grounder = MiraEmbeddingsGrounder(ontologyFilePath, None, lambda, alpha)
+          val ontologyFilePath = domainConfig.getString("ontologyPath")
```

Comment:
At some future point we might collect all of these variables into a structure and read them all at once.  There's a relatively easy way to do this.